src/components/QuestionModal.jsx

```diff
@@ -51,14 +48,14 @@ const QuestionModal = props => {
 
       {showingAnswer && (
         <div className="answer-buttons">
-          <button onClick={handleScoreCorrect} className="hell-yeah-button">
+          <button onClick={() => handleScore(true)} className="hell-yeah-button">
             HELL YEAH!{" "}
             <span role="img" aria-label="red cross">
               ✅
             </span>
           </button>
 
-          <button onClick={handleScoreWrong} className="close-modal-button">
+          <button onClick={handleScore} className="close-modal-button">
```

Comment:
For consistency and clarity, and considering the suggested changes above, I think explicitly passing `false` here would be a better choice.